agenda/helpers.py

```diff
@@ -77,6 +77,8 @@ def calendar_events(request, pk):
     events = []
     multiband = len(user_assocs) > 1
     for g in the_gigs:
+        if (g.is_in_trash):
+            continue
```

Comment:
Awesome! Thanks so much for taking this on!

The logic of your fix looks correct, but rather than check each gig as we print them, how about moving this up to the filter line, which is where we pull data out of the database. Perhaps something like `is_in_trash=False` around like 74?

Reply:
ah yeah that would be way better, I tunnel visioned on the filtering logic to figure out how all these models worked. Will update with a change to query instead.

Reply:
OK - addressed the fix with a new commit, but in doing so I unintentionally created two new commits - one adding the filter to the query and a second removing the filter after the query - still getting used to the GitHub tooling / flows. LMK if its possible to squash these commits on merge, or otherwise I can recreate this as a single commit on a separate branch and re-create the pull request.

Reply:
No problem, I can squash when I merge. Thanks!